### DISCUSSION TITLE
`next-extraction-plugin` package's README is unclear

### DISCUSSION THREAD

**Discussion by yume-chan** [microsoft/griffel#190] [Q&A] [answered]
1. It skips the steps for install `@griffel/webpack-loader` (and maybe other dependencies)
2. The code example contains `require('@griffel/webpack-extraction-plugin')`, should be `require('@griffel/next-extraction-plugin')`
3. Not clear how to use with TypeScript project. With only the configuration in README, `next dev` throws this error:

```text
error - ./src/components/common/portal.tsx:6:7
Syntax error: Support for the experimental syntax 'flow' isn't currently enabled (6:8):

  4 | import { LuminanceShifter } from "../theme";
  5 |
> 6 | export interface PortalContextValue {
    |        ^
  7 |   target: HTMLDivElement | null;
  8 | }
  9 |

Add @babel/preset-flow (https://github.com/babel/babel/tree/main/packages/babel-preset-flow) to the 'presets' section of your Babel config to enable transformation.
If you want to leave it as-is, add @babel/plugin-syntax-flow (https://github.com/babel/babel/tree/main/packages/babel-plugin-syntax-flow) to the 'plugins' section to enable parsing.
```

Tried multiple combination with `next/babel`, `@babel/preset-typescript`, `@babel/preset-react` presets but still can't get it to work.

**Comment by layershifter** [comment, marked as answer]
@yume-chan it's a good feedback, thank you 👍 I created #194 to make updates, can you please check it?
